app/grants/admin.py

```diff
@@ -28,7 +28,7 @@
 import twitter
 from grants.models import (
     CartActivity, CLRMatch, Contribution, Flag, Grant, GrantCategory, GrantCLR, GrantCollection, GrantStat, GrantType,
-    MatchPledge, PhantomFunding, Subscription,
+    MatchPledge, PhantomFunding, Subscription, GrantBrandingRoutingPolicy,
```

Comment:
@aamustapha make sure to run the test suite before you mark a PR ready for review (or make sure Travis CI passes). Here you will run into an isort error because the python style checker assumes the imports are done in alphabetical order